mifosng-android/src/main/res/navigation/nav_graph.xml

```diff
@@ -74,6 +74,39 @@
             app:argType="integer" />
     </activity>
 
+    <fragment
+        android:id="@+id/checkerInboxTasksFragment"
+        android:name="com.mifos.mifosxdroid.online.checkerinbox.CheckerInboxTasksFragment"
+        android:label="CheckerInboxTasksFragment">
+        <action
+            android:id="@+id/action_checkerInboxTasksFragment_to_checkerInboxFragment"
+            app:destination="@id/checkerInboxFragment" />
+    </fragment>
+
+
+    <fragment
+        android:id="@+id/checkerInboxFragment"
+        android:name="com.mifos.mifosxdroid.online.checkerinbox.CheckerInboxFragment"
+        android:label="CheckerInboxFragment" />
+
+
+    <fragment
+        android:id="@+id/individualCollectionSheetFragment"
+        android:name="com.mifos.mifosxdroid.online.collectionsheetindividual.IndividualCollectionSheetFragment"
+        android:label="IndividualCollectionSheetFragment">
+
+        <action
+            android:id="@+id/action_individualCollectionSheetFragment_to_individualCollectionSheetDetailsFragment"
+            app:destination="@id/individualCollectionSheetDetailsFragment" />
+
+    </fragment>
+
+
+    <fragment
+        android:id="@+id/generateCollectionSheetFragment"
+        android:name="com.mifos.mifosxdroid.online.generatecollectionsheet.GenerateCollectionSheetFragment"
+        android:label="GenerateCollectionSheetFragment" />
+
```

Comment:
Why are we adding this again? Wasn't it already removed during the previous cleanup process?